src/utils/clack-utils.ts

```diff
@@ -30,6 +31,42 @@ interface WizardProjectData {
   projects: SentryProjectData[];
 }
 
+export interface CliSetupConfig {
+  filename: string;
+  name: string;
+
+  likelyAlreadyHasAuthToken(contents: string): boolean;
+  tokenContent(authToken: string): string;
+
+  likelyAlreadyHasOrgAndProject(contents: string): boolean;
+  orgAndProjContent(org: string, project: string): string;
+}
+
+export class SourceMapsCliSetupConfig implements CliSetupConfig {
```

Comment:
m-h: I like the idea of the interface but I'd prefer to create objects instead of classes for the concrete implementations. We want to keep a functional approach in the `clack`-based (new) wizards and avoid using classes. 

Suggestion:
* Keep the `CliSetupConfig` interface
* Create objects for the concrete implementations that satisfy the interface
  * Either directly or by writing a `make[Concrete]CliSetupConfig` helper
 
Wdyt?

Reply:
sounds good, I was just not sure which approach is better. Will change that 👍

Reply:
Addressed in https://github.com/getsentry/sentry-wizard/pull/414/commits/4473bd5cee80a36300e703769a5ae240172eff60